src/PayID/pay-id-client.ts

```diff
@@ -197,10 +195,10 @@ export default class PayIDClient {
 
     const travelRule = new TravelRule(originator, beneficiary)
 
-    const compliance = new Compliance(complianceType, travelRule)
+    const compliance = new Compliance(ComplianceType.TravelRule, travelRule)
 
     const signatureWrapper = new SignatureWrapperCompliance(
-      complianceType,
+      'Compliance',
```

Comment:
Should this not be 'TravelRule'?

Reply:
I'm told by @dino-rodriguez  and @tedkalaw that it should be TravelRule.

Reply:
I did not look carefully enough at this and realize we were building a `SignatureWrapper`.